android/filament-android/src/main/java/com/google/android/filament/View.java

```diff
@@ -171,6 +171,14 @@
          */
         @NonNull
         public QualityLevel quality = QualityLevel.LOW;
+
+        /**
+         * The upsampling setting controls the quality of the ambient occlusion buffer upsampling.
+         * The default is QualityLevel.LOW and uses bilinear filtering, a value of
```

Comment:
What does MEDIUM do?

Reply:
Nothing for now. I left it open because we could maybe have LOW==NEAREST.

Reply:
What do you mean by nothing? You don't get SSAO or do you get the equivalent of LOW or the equivalent of HIGH?

Reply:
It's like LOW.

Reply:
Well, you should have the documentation explain that then :p